### DISCUSSION TITLE
Where models are stored?

### DISCUSSION THREAD

**Discussion by lehcode** [nicobrauchtgit/obsidian-Smart2Brain#100] [General]
I am wondering where plugins saves downloaded LLMs..

Thanks

**Comment by Leo310** [comment]
We don't save them as they are handled by Ollama. For more details take a look at https://github.com/ollama/ollama/blob/main/docs/faq.md#where-are-models-stored

**Reply by lehcode** [reply]
Yep 👌Thanks anyway!
I already figured out that they're under ollama directory.
